internal/orchestrion/_integration/go.mod

```diff
@@ -8,7 +8,7 @@ require (
 	cloud.google.com/go/pubsub v1.44.0
 	github.com/99designs/gqlgen v0.17.62
 	github.com/DataDog/go-libddwaf/v3 v3.5.1
-	github.com/DataDog/orchestrion v1.0.3-rc.1.0.20250120134012-8c2a6e2c1d70
+	github.com/DataDog/orchestrion v1.0.4
```

Comment:
It needs to be a commitish on `main` for now as this expects to work with the upcoming `1.1.x` release line (that we are yet to tag).